src/app/map/Cache/Handlers/navigation-handler.ts

```
@@ -32,24 +34,72 @@
 }
 
 export function createNavigationHandler(config: NavigationHandlerConfig) {
-  const { dispatch, getState, dataHandler } = config;
+  const { dispatch, getState, dataHandler, eventBus } = config;
 
   const navigateToItem = async (
-    itemCoordId: string,
+    itemDbId: string,
     options: NavigationOptions = {},
   ): Promise<NavigationResult> => {
     const { } = options; // Options reserved for future use
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)